dist/rpm/scion.postinst

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+# Create system user
+adduser --system --user-group --create-home --home /var/lib/scion scion
```

Comment:
Since this is a system user, I believe it should not have a home directory and have no login shell.
The `/var/lib/scion` cache file directory can be created together with the `/etc/scion` directory below.
Being explicit about the group name ensures that if the user was successfully created, the directory ownership we set later also match, and no `USERGROUPS_ENAB` config interfered with our expected gid.

Reply:
done